python/python/glide/async_commands/core.py

```diff
@@ -205,37 +209,96 @@ class TrimByMinId(StreamTrimOptions):
     Stream trim option to trim by minimum ID.
     """
 
-    def __init__(self, exact: bool, threshold: str, limit: Optional[int] = None):
+    def __init__(
+        self,
+        threshold: str,
+        method: str,
+        exact: Optional[bool] = None,
+        limit: Optional[int] = None,
+    ):
+        super().__init__(threshold, method, exact, limit)
+
+    @classmethod
+    def create(cls, threshold: str):
+        """
+        Initialize trim option by minimum ID.
+
+        Args:
+            threshold (str): Threshold for trimming by minimum ID.
+        """
+        return cls(threshold, "MINID", None, None)
+
+    @classmethod
+    def create_withExact(cls, exact: bool, threshold: str):
         """
         Initialize trim option by minimum ID.
 
         Args:
             exact (bool): If `true`, the stream will be trimmed exactly.
                 Otherwise the stream will be trimmed in a near-exact manner, which is more efficient.
             threshold (str): Threshold for trimming by minimum ID.
-            limit (Optional[int]): Max number of entries to be trimmed. Defaults to None.
-                Note: If `exact` is set to `True`, `limit` cannot be specified.
         """
-        super().__init__(exact, threshold, "MINID", limit)
+        print(threshold)
+        return cls(threshold, "MINID", exact, None)
+
+    @classmethod
+    def create_withLimit(cls, threshold: str, limit: int):
```

Comment:
Is there a precedent for using camelCase midway through a function name in Python? I thought every method was supposed to be snake_case. I'm okay with it being this way if we're consistent about it, but just want to be sure.

Reply:
It odd.  They use snake_case but are inconsistent with underscores, like `zrange_withscores`.  
I will update this to `create_withlimit` and see what Shoham things.